ManatiCR/artavia
# AR-22 equipo page #2

[AR-22](https://manati.atlassian.net/browse/AR-22)
[Figama designs](https://www.figma.com/file/sRoXqR6hyiNvc8moaFeZA6/Artavia-Consultor%C3%ADa)
-
  This PR depends of this one: https://github.com/ManatiCR/artavia/pull/4
---
- Create the list of personas, equipo page.
- Page to test: http://artavia.docker/es/nuestro-equipo.
**Steps to test**
- Create content in the following order: clientes, personas, colaboradores exteriores (use devel generate)
- You should be able to add an outstanding member (node persona) to the page, to do it go to the entityque miembro destacado and added the node you one to be outstanding (http://artavia.docker/es/admin/structure/entityqueue/miembro_destacado/miembro_destacado)
- The outstanding member should be first in the equipo page and appear just once.
---
- There should not be a pager
